Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -339,20 +339,10 @@ protected Conditions createConditions(final CallbackProperties properties) {
             // Create resource for Authentication Decision Statement
             final String resource = properties.getAuthorizationResource();
 
-            // Options are Permit, Deny and Indeterminate
-            String decision = properties.getAuthorizationDecision();
-            if (decision == null) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
-            if (!isValidAuthorizationDecision(decision)) {
-                decision = AUTHZ_DECISION_PERMIT;
-            }
-
             final Evidence evidence = createEvidence(properties, subject);
```

Comment:
Remove the isValidAuthorizationDecision and AUTHZ_DECISION_DENY method in SAMLAssertionbuilder as well.